src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    sum = days * 40;
+  }
+
+  if (days >= 3) {
+    sum = days * 40 - 20;
+  }
+
+  if (days >= 7) {
```

Comment:
It will be nice if instead 40 you will use a variable (in the case if this price is changed, it will be easier to change it only in 1 place)